runtime/vm/createramclass.cpp

```diff
@@ -32,6 +32,9 @@
 #include "../util/ut_module.h"
 #undef UT_MODULE_LOADED
 #undef UT_MODULE_UNLOADED
+#include "../shared_common/AbstractMemoryPermission.hpp"
+#undef UT_MODULE_LOADED
+#undef UT_MODULE_UNLOADED
```

Comment:
What is this for?

Reply:
This is so I could use the `ROUND_UP_TO` macro. Is there a better way for me to access this macro from `AbstractMemoryPermission.hpp`

Reply:
Use the round up to macro in j9.h (`ROUND_UP_TO_POWEROF2`). That file is already included so you dont need to add any includes.

Reply:
Agreed.